app/services/report_forwarding_service.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class ReportForwardingService < BaseService
+  include Payloadable
+
+  def call(report, forwarder, options = {})
```

Comment:
Arguably we could change this signature to:
```suggestion
  def call(report, forwarder, forward_to_domains = [])
```

The options usage is just from moving code across from ReportService